readme.markdown

```diff
@@ -117,6 +119,13 @@ default `opts` values:
             return cb(err);
         });
     },
+    unwrapSymlink: function unwrapSymlink(file, cb) {
```

Comment:
I'm not too fond of this name - I just copied the name from the code. Should we call it `realpath` and `realpathSync` to mirror `readFile` and `readFileSync` options?

Reply:
i don't have a strong opinion; "realpath" is definitely the typical name for this, and "unwrap" really is just a word that makes sense in my head, not something that actually makes sense :-)

I'll update this to match your suggestions.